pytorch_lightning/utilities/cli_registries.py

```diff
@@ -0,0 +1,79 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
We have `plugins_registry` so this should be `cli_registry`.

Also, it should be put somewhere different than the `utilities` directory.

Reply:
cli.py is part of utilities. I moved the code back within that file.